src/Sulu/Bundle/AdminBundle/Resources/js/containers/Application/variables.scss

```diff
@@ -1 +1,3 @@
 $viewPadding: 60px;
+$toolbarHeight: 60px;
```

Comment:
There is already a `$toolbarHeight` variable in `toolbar.scss`. Maybe we should move that one to a `variable.scss` in the `Toolbar` component, and import that `variable.scss` file wherever we need it (also in the places we already using it currently), because otherwise the CSS classes are imported as well.